sdk/core/Azure.Core.Amqp/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.1.0-beta.2 (Unreleased)
 
+### Added
+- All section properties of the `AmqpAnnotatedMessage` are now lazily allocated to reflect that they are defined as optional in the AMQP specification, section 3.2.
```

Comment:
The section properties were already lazily allocated, right? It is just now that we can actually leverage this fact since we have the HasSection method.